recipes/docker.rb

```diff
@@ -216,6 +216,28 @@
             })
 end
 
+cookbook_file node['hops']['docker']['hopsfsmount-seccomp-profile'] do
+  source 'hopsfsmount_seccomp_profile.json'
+  owner 'root'
+  mode '0755'
+  action :create
+end
+
+hopfsmount_apparmor_profile="/etc/apparmor.d/#{node['hops']['docker']['hopfsmount-apparmor-profile']}"
+cookbook_file hopfsmount_apparmor_profile do
+  source 'hopsfsmount_apparmor_profile'
+  owner 'root'
+  mode '0755'
+  action :create
```

Comment:
Can I just move this attribute to this cookbook?

Reply:
I thought we should still execute it even though the contents manager is initially put as hdfscontents so that they can still be able to change it later to hopsfsmount.

Reply:
@gibchikafa I missed your second last comment. I think it logically belongs to hopsworks-chef